CHANGELOG.md

```diff
@@ -26,6 +27,54 @@
  - [0.0.5](#005---20180602)
  - [0.0.4 and prior](#004-and-prior)
 
+## [1.2.0] - 2021/03/22
+
+#### Added
+
+- Ingresses now support `konghq.com/request-buffering` and
+  `konghq.com/response-buffering` annotations, which set the
+  `request-buffering` and `response-buffering` settings on associated Kong
+  routes.
+  [#1016](https://github.com/Kong/kubernetes-ingress-controller/pull/1016/)
+- Added `--dump-config` flag to dump generated Kong configuration to a
+  temporary file to debug issues where the controller generates unexpected
+  configuration or unacceptable configuration. When set to `enabled` it redacts
+  sensitive values (credentials and certificate keys), and when set to
+  `sensitive`, it includes all configuration.
+  [#991](https://github.com/Kong/kubernetes-ingress-controller/pull/991/)
+- Added support for mtls-auth plugin credentials (requires Enterprise 2.3.2.0
+  or newer).
+  [#1078](https://github.com/Kong/kubernetes-ingress-controller/pull/1078/)
+- The KongClusterPlugin CRD is now optional, for installation in clusters
+  where KIC administrators do not have cluster-wide permissions.
+
+#### Fixed
+
+- The admission webhook can now validate KongPlugin configurations stored in a
+  Secret.
+  [#1036](https://github.com/Kong/kubernetes-ingress-controller/pull/1036/)
+
+#### Under the hood
+
+- Build configuration allows target architectures other than `amd64`. Note that
+  other architectures are not officially supported.
+  [#1046](https://github.com/Kong/kubernetes-ingress-controller/pull/1046/)
+- Updated to Go 1.16. Make sure to update your Go version if you build your own
+  controller binaries.
+  [#1110](https://github.com/Kong/kubernetes-ingress-controller/pull/1110/)
+- Refactored synchronization loop into more discrete components and created
+  packages for them.
+  [#1027](https://github.com/Kong/kubernetes-ingress-controller/pull/1027/)
+  [#1029](https://github.com/Kong/kubernetes-ingress-controller/pull/1029/)
+- Began work on
+  [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder)-based
+  implementation of the controller.
```

Comment:
```suggestion
- Broad refactoring (with the purpose of exposing KIC's logic as libraries), in preparation for an architectural upgrade of KIC to a [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder)-based
  implementation of the controller (expected to be released as KIC v2.0).
```

I thought we should make it clear that kubebuilder is not coming in this release, and what really affects v1.2.0 is the refactoring.